src/tools/compiletest/src/ferrocene_annotations.rs

```diff
@@ -115,7 +116,9 @@ impl Collector {
     fn collect_annotations(&self, path: &Path, contents: &str) -> Vec<Annotation> {
         let mut found = Vec::new();
         for line in contents.lines() {
-            let prefix = if path.file_name() == Some(OsStr::new("Makefile")) {
+            let prefix = if path.file_name() == Some(OsStr::new("Makefile"))
+                || path.file_name() == Some(OsStr::new("rmake.rs"))
```

Comment:
I believe this is an incorrect assumption. The `rmake.rs` file is Rust and it's comments are prefixed with `//`.